python/lsst/summit/extras/focusAnalysis.py

```diff
@@ -90,18 +93,18 @@ class SpectralFocusAnalyzer:
     focusAnalyzer.run() can be used instead of the last two lines separately.
     """
 
-    def __init__(self, embargo=False):
+    def __init__(self, embargo: bool = False):
         self.butler = makeDefaultLatissButler(embargo=embargo)
         self._bestEffort = BestEffortIsr(embargo=embargo)
         qfmTaskConfig = QuickFrameMeasurementTaskConfig()
         self._quickMeasure = QuickFrameMeasurementTask(config=qfmTaskConfig)
 
         self.spectrumHalfWidth = 100
         self.spectrumBoxLength = 20
-        self._spectrumBoxOffsets = [882, 1170, 1467]
+        self._spectrumBoxOffsets = [882.0, 1170.0, 1467.0]
         self._setColors(len(self._spectrumBoxOffsets))
 
-    def setSpectrumBoxOffsets(self, offsets):
+    def setSpectrumBoxOffsets(self, offsets: List[float]) -> None:
         """Set the current spectrum slice offsets.
```

Comment:
These actually are integers, and need to be typed as such, as they're used for de-referencing arrays (or the equivalent, actually used in Box2I construction, but it's the same thing.)

Reply:
OK. In this case I was just following the docstring:

        offsets : `list` of `float`